src/vm/dllimport.cpp

```diff
@@ -4194,9 +4194,13 @@ static void CreateNDirectStubWorker(StubState*         pss,
     bool fHasCopyCtorArgs = false;
     bool fStubNeedsCOM = SF_IsCOMStub(dwStubFlags);
 
+#if defined(_TARGET_X86_) || defined(_TARGET_ARM_)
```

Comment:
ARM32 uses the same jit as x86 so I am not sure if that would not have the same problem. It is also not easy to do testing for arm32 right now. So I will leave it the way it is right now.